### PR TITLE
Feature/branch sarahs working copy

### PR DESCRIPTION
**Pull request by sarahlliza**
Updating David's branch

### CODE REVIEW

File: css/main-css.css [hunk 1]
```diff
@@ -49,12 +49,11 @@ h5 {
 
 body{
     width: 100%;
-	background: url('../images/watercolour-header.png') no-repeat center center fixed;
+	background: url('../images/watercolour_header_large.png') no-repeat center center;
```
Comment (david-leonard) [inline css/main-css.css:52]: we should actually change the naming convention here

File: css/main-css.css [hunk 2]
```diff
@@ -144,8 +144,14 @@ body{
...
 
 /* Responsive layout - makes the three columns stack on top of each other instead of next to each other */
-@media screen and (max-width: 600px) {
+@media screen and (max-width: 640px) {
     .column {
         width: 100%;
     }
+}
+
+@media (max-width: 1035px) {
```
Comment (david-leonard) [inline css/main-css.css:153]: did you test this?

File: css/main-css.css (hunk 1 above)
Comment (sarahlliza) [reply css/main-css.css:52]: No I don't think so

File: css/main-css.css (hunk 2 above)
Comment (sarahlliza) [reply css/main-css.css:153]: Never

**Pull request merged**